.github/workflows/pr_build_images.yaml

```diff
@@ -27,7 +27,12 @@ jobs:
       - uses: actions/checkout@v2
       - name: Build the images
         run: |
-          wget https://github.com/just-containers/s6-overlay/releases/download/v1.22.1.0/s6-overlay-amd64.tar.gz
+          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.1.2/s6-overlay-x86_64.tar.xz
+          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.1.2/s6-overlay-noarch.tar.xz
+          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.1.2/s6-overlay-symlinks-arch.tar.xz
+          wget https://github.com/just-containers/s6-overlay/releases/download/v3.1.1.2/s6-overlay-symlinks-noarch.tar.xz
+          unxz s6-overlay*.tar.xz
+          gzip s6-overlay*.tar
```

Comment:
Does tar not support xz in the container?

Reply:
No, and I'd rather save container size than install it.